From f7d62caaa2470efbe7577fbe988b4317e8d7e6bd Mon Sep 17 00:00:00 2001
From: phhusson <phh@phh.me>
Date: Mon, 7 Jan 2019 21:13:46 +0100
Subject: [PATCH 1/2] Set ro.build.fingerprint in system/etc/prop.default

Change-Id: I51ca98c608e30b134640c42ea5d2adf8364f9c52
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 9a16e83d4..22e374eb8 100644
--- a/Makefile
+++ b/Makefile
@@ -116,6 +116,7 @@ $(INSTALLED_DEFAULT_PROP_TARGET): $(intermediate_system_build_prop)
 	$(hide) echo ro.bootimage.build.date=`$(DATE_FROM_FILE)`>>$@
 	$(hide) echo ro.bootimage.build.date.utc=`$(DATE_FROM_FILE) +%s`>>$@
 	$(hide) echo ro.bootimage.build.fingerprint="$(BUILD_FINGERPRINT_FROM_FILE)">>$@
+	$(hide) echo ro.build.fingerprint="$(BUILD_FINGERPRINT_FROM_FILE)">>$@
 	$(hide) build/tools/post_process_props.py $@
 ifdef property_overrides_split_enabled
 	$(hide) mkdir -p $(TARGET_ROOT_OUT)
-- 
2.19.1

